business_objects/personal_access_token.py

```diff
@@ -0,0 +1,83 @@
+import datetime
+from typing import List
+from ..session import session
+from submodules.model.business_objects import general
+from submodules.model.models import PersonalAccessToken
+from datetime import date
+
+
+def get(project_id: str, user_id: str, name: str) -> PersonalAccessToken:
+    return (
+        session.query(PersonalAccessToken)
+        .filter(
+            PersonalAccessToken.project_id == project_id,
+            PersonalAccessToken.user_id == user_id,
+            PersonalAccessToken.name == name,
+        )
+        .first()
+    )
+
+
+def get_all(project_id: str) -> List[PersonalAccessToken]:
+    return (
+        session.query(PersonalAccessToken)
+        .filter(PersonalAccessToken.project_id == project_id)
+        .all()
+    )
+
+
+def get_by_token(project_id: str, token: str) -> PersonalAccessToken:
```

Comment:
i think this should be something like get by token_hash to signal that not the actual but the hashed version is compared. though I'm unsure since the column is called token :D

Reply:
I understand that point but i would leave it as it is tbh since in the gateway it should be clear that the token is hashed for get requests. But of course could get changed

Reply:
works for me